cmd/queue/main.go

```diff
@@ -331,7 +326,7 @@ func main() {
 		logger.Fatalw("Failed to create stats reporter", zap.Error(err))
 	}
 
-	statChan := make(chan *autoscaler.Stat, statReportingQueueLength)
+	statChan := make(chan *autoscaler.Stat)
 	defer close(statChan)
 	go func() {
 		for s := range statChan {
```

Comment:
should the report be in a goroutine?

Reply:
See description. I don‘t think it‘s necessary. It could though.